control/veda_file_ingest.py

```diff
@@ -162,7 +162,7 @@ def queue_job(self):
             jobid = uuid.uuid1().hex[0:10]
             celeryapp.worker_task_fire.apply_async(
                 (veda_id, encode_profile, jobid),
-                queue=self.auth_dict['celery_worker_queue']
+                queue=self.auth_dict['celery_worker_queue'].split(',')[0]
```

Comment:
@yro why are we doing this?

Reply:
There were/are two queues, which are read correctly when instantiating a celery worker as two queues, but when enqueueing it's read as a continuous string. Since we're only using the one queue for now, this is basically a hack until we deprecate the other queue completely.